pkg/reconcilers/imagestreams.go

```diff
@@ -0,0 +1,66 @@
+package reconcilers
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/3scale/3scale-operator/pkg/common"
+	imagev1 "github.com/openshift/api/image/v1"
+)
+
+func GenericImagestreamMutator(existingObj, desiredObj common.KubernetesObject) (bool, error) {
```

Comment:
nitpick. `GenericImagestreamMutator``should be better called `GenericImageStreamMutator` (Uppercase S in stream)

Reply:
:+1: